Expensify/App
# Make LHN rows look the same as chatswitcher ones

@shawnborton, can you review (although I have reused the ChatSwitcher file, so the styles should be fine)?
@marcaaron, can you review? Any reason we might want to keep these two in separate files?   

Instead of applying the styles to `SideBarLink.js`, I decided, instead, to use `ChatSwitcherRow` (now renamed to `ChatLinkRow` in both places, since we'll likely always want to keep the same format in both places and it just required a few changes.

### Fixed Issues
Fixes https://github.com/Expensify/Expensify/issues/144292

### Tests/QA
1. Open Native Chat and open a few chats in the LHN (you might need to pin them). Single-participant DMs should show the icon, the name of the person (if they have one set) and the login below. Reports should show the report's name and group DMs, a list of the participants' names (basically, the same as the chat switcher):
<img width="300" alt="Screen Shot 2020-11-12 at 1 11 56 PM" src="https://user-images.githubusercontent.com/19366280/99060110-d7792280-259f-11eb-842f-da6c90cd65d1.png">
2. Make sure receiving a message correctly bolds the row in the LHN:
<img width="299" alt="Screen Shot 2020-11-16 at 10 31 13 AM" src="https://user-images.githubusercontent.com/19366280/99236033-f2919f80-27f6-11eb-949c-23b797f5fe34.png">
3. Make sure ChatSwitcher still works correctly.
4. Check that 1-3 are all true across platforms
